README.md

```diff
@@ -98,10 +98,10 @@ System property hiveconf_hive.execution.engine to 'tez'.
         </plugin>
 
 ### Timeout
-It's possible to configure HiveRunner to make tests time out after some time and retry those tests a couple of times. This is to cover for the bug
+It's possible to configure HiveRunner to make tests time out after some time and retry those tests a couple of times, but only when using `StandaloneHiveRunner` (this is not available in the `HiveRunnerExtension` (from HiveRunner 5.x and up). This is to cover for the bug
```

Comment:
(this is not available in the `HiveRunnerExtension` (from HiveRunner 5.x and up).
->
this is not available in the `HiveRunnerExtension` (from HiveRunner 5.x and up).